core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -167,8 +167,8 @@ oppia.factory('threadDataService', [
         payload.commit_message = commitMsg;
       }
       _openThreadsCount -= 1;
-      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(
-        onSuccess).error(function() {
+      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).then(
+        onSuccess, function() {
```

Comment:
ThreadDataService.resolveSuggestion is being called in FeedbackTab and it doesn't expect an object to be passed. Here's the usage:
`threadDataService.resolveSuggestion(
          $scope.activeThread.thread_id, result.action, result.commitMessage,
          function() {...}, function(){...}` 
We should probably make a note (linting?) that from on .then should be used and the parameter is a response instead of the data object.